code/datalab-app/src/app/components/notebooks/CreateNotebookForm.js

```diff
@@ -6,21 +6,49 @@ import { renderTextField, renderTextArea, renderSelectField, renderAdornedTextFi
 import { syncValidate, asyncValidate } from './newNotebookFormValidator';
 import { ANALYSIS, getStackSelections } from '../../../shared/stackTypes';
 
+const fieldStyle = { minWidth: 250 };
```

Comment:
Probably better to try to make this sort of change in a different branch as it is unrelated.